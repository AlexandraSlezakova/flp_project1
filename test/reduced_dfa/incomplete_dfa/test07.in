0,1,2,3
cab
0
3
0,a,1
0,b,2
1,c,3
2,c,3
3,c,3
