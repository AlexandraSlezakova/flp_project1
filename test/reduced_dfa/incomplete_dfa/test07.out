0,1,2
abc
0
2
0,a,1
0,b,1
1,c,2
2,c,2
